src/lexer.c

```diff
@@ -2353,10 +2351,10 @@ void Lexer::poundLine()
                 continue;                       // skip white space
 
             case '_':
-                if (mod && memcmp(p, "__FILE__", 8) == 0)
```

Comment:
I just hope it doesn't introduce segfaults when printing `__FILE__` when a module declaration is missing (I remember a similar bug with `__MODULE__`).